ipywidgets/widgets/widget.py

```diff
@@ -708,7 +708,6 @@ def _ipython_display_(self, **kwargs):
             # http://www.iana.org/assignments/media-types/media-types.xhtml.
             data = {
                 'text/plain': repr(self),
```

Comment:
If we want to ensure this never gets too big, how about `repr(self)[:160]` (or some other appropriate number).

Reply:
I think it is nice to have the full repr available, I'm just arguing whether that should be the fallback format or not.

Reply:
Since you can always do `repr()` to get the full repr, I think chopping it off (and appending an ellipsis) is a good compromise for the fallback.